test/forward_engineering/mappers/directives.spec.js

```diff
@@ -1,6 +1,6 @@
 const { describe, it } = require('node:test');
 const assert = require('node:assert');
-const { getDirectivesUsageStatement } = require('../../forward_engineering/mappers/directives');
+const { getDirectivesUsageStatement } = require('../../../forward_engineering/mappers/directives');
```

Comment:
moved this file to the proper folder according to real structure inside `forward_engineering` folder